test/e2e/knit_snapproc_cpulist.go

```diff
@@ -0,0 +1,67 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/jaypipes/ghw/pkg/snapshot"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+)
+
+const (
+	snapprocName = "snapproc.tgz"
+)
+
+var _ = g.Describe("knit snapproc tests", func() {
+	g.Context("With process snapshot", func() {
+		g.It("Should be readable by cpulist", func() {
+			cmdline := []string{
+				filepath.Join(binariesPath, "knit"),
+				"snapproc",
+			}
+			fmt.Fprintf(g.GinkgoWriter, "running: %v\n", cmdline)
+
+			cmd := exec.Command(cmdline[0], cmdline[1:]...)
+			cmd.Stderr = g.GinkgoWriter
+			_, err := cmd.Output()
+			o.Expect(err).ToNot(o.HaveOccurred())
```

Comment:
it possible the test will fail here, in such case you will not remove the `snapprocName` file, please run clean under the AfterEach

Reply:
good point, fixed.